qiskit/providers/aer/primitives/estimator.py

```diff
@@ -0,0 +1,307 @@
+        if isinstance(circuits, QuantumCircuit):
+            circuits = (circuits,)
+        circuits = tuple(init_circuit(circuit) for circuit in circuits)
+
+        if isinstance(observables, (PauliSumOp, BaseOperator)):
+            observables = (observables,)
+        observables = tuple(
+            init_observable(observable).simplify(atol=0) for observable in observables
+        )
+
+        super().__init__(
+            circuits=circuits,
+            observables=observables,
+            parameters=parameters,
+        )
+        self._is_closed = False
+        backend_options = {} if backend_options is None else backend_options
+        method = (
+            "density_matrix"
+            if approximation and "noise_model" in backend_options
+            else "statevector"
+        )
+        self._backend = AerSimulator(method=method)
+        self._backend.set_options(**backend_options)
+        self._transpile_options = Options()
+        if transpile_options is not None:
+            self._transpile_options.update_options(**transpile_options)
+        self.approximation = approximation
+
+    def _call(
+        self,
+        circuits: Sequence[int],
+        observables: Sequence[int],
+        parameter_values: Sequence[Sequence[float]],
+        **run_options,
+    ) -> EstimatorResult:
+        if self._is_closed:
+            raise QiskitError("The primitive has been closed.")
+
+        seed = run_options.pop("seed", None)
+        if seed is not None:
+            run_options.setdefault("seed_simulator", seed)
+
+        if self.approximation:
+            shots = run_options.pop("shots", None)
+
+            experiments = []
+            parameter_binds = []
+            for i, j, value in zip(circuits, observables, parameter_values):
+                if len(value) != len(self._parameters[i]):
+                    raise QiskitError(
+                        f"The number of values ({len(value)}) does not match "
+                        f"the number of parameters ({len(self._parameters[i])})."
+                    )
+
+                circuit = self._circuits[i].copy()
+                observable = self._observables[j]
+                if circuit.num_qubits != observable.num_qubits:
+                    raise QiskitError(
+                        f"The number of qubits of a circuit ({circuit.num_qubits}) does not match "
+                        f"the number of qubits of a observable ({observable.num_qubits})."
+                    )
+                if shots is None:
+                    circuit.save_expectation_value(observable, range(circuit.num_qubits))
+                else:
+                    circuit.save_expectation_value_variance(observable, range(circuit.num_qubits))
+                experiments.append(circuit)
+                parameter_binds.append({k: [v] for k, v in zip(self._parameters[i], value)})
+            experiments = transpile(experiments, self._backend, **self._transpile_options.__dict__)
```

Comment:
do we need `transpile` always?

Reply:
I'm not sure if `transpile` can be skipped because I don't understand the `transpile` spec...

If the `backend` doesn't have a noise_model and basis_gate, can it be skipped?

Reply:
Or, can I introduce `skip_transpilation` option?

Reply:
OK, I added the cache in ed393a2.